docs/03-assertions.md

```diff
@@ -10,7 +10,15 @@ test('unicorns are truthy', t => {
 });
 ```
 
-Assertions are bound to their test so you can assign them to a variable or pass them around:
+**Note**: Unlike other assertion libraries, AVA assertions do ***not***
+stop the test when they fail.
+
+When a test has multiple assertion failures,
+AVA only displays the *first* failure.
+There's currently no way to see all failures from a test
+([see issue #261](https://github.com/avajs/ava/issues/261)).
```

Comment:
This should come at the end of the section, as it was before. The section starts with how you can access the assertions and how they're bound. An explanation of their runtime behavior shouldn't be sandwiched in between.

As for the text itself, what do you think about this?

```suggestion
AVA's assertions do not throw errors. Your test will continue executing after an assertion failure. AVA only displays the *first* failure.
```